src/layout/utils.ts

```diff
@@ -21,10 +21,44 @@ export function measureText(text: string) {
   return result;
 }
 
+function parsePadding(padding?: number[]) {
```

Comment:
We should write a unit test for this ( can be in a follow up PR ).

Reply:
Yes, within a follow up PR sounds good. Have you already experience how we could write Unit tests to check properties of SVGs? I am specifically asking, because the padding does not seem to be trackable like a CSS property and intuitively I dont know how to test this feature.

Reply:
I would make the unit test just test the input/output of this pure function. No need to test the SVG props.

Reply:
Makes totally sense, I've just added 3 test cases for the possible input types in dcebff5.